e2e/tests/api-driven/src/demo-workspace/steps/verification_steps.ts

```diff
@@ -83,18 +83,20 @@ Then<CustomWorld>("I should be able to see a flow", async function (this) {
   );
 });
 
-Then<CustomWorld>(
-  "I should be able to {string} the flow",
```

Comment:
Two issues I had to fix here - 
 - Async/await wasn't being handled correctly (thus allowing `undefined` to pass) - `hasSucceeded` is always truthy.
 - `updateFlow` changes the flow slug, and `deleteFlow` tries to get it by it's _original_ value - it will always fail and not try to delete anything. To resolve this I'm now tracking the ID.